packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -402,6 +452,16 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, ERC900,
         _setTotalActiveBalanceLimit(_totalActiveBalanceLimit);
     }
 
+    /**
+    * @notice `_allowed ? 'Allow' : 'Disallow'` `_lockManager` as a lock manager
+    * @param _lockManager Address of the lock manager to be changed
+    * @param _allowed Whether the lock manager is allowed
+    */
+    function changeLockManager(address _lockManager, bool _allowed) external onlyConfigGovernor {
```

Comment:
> I'm also wondering the tradeoff between keeping this configuration here, vs. in the Protocol config. I assume the ability to persist this whitelist over individual registry upgrades wouldn't be worth the extra gas cost and complexity?

I'd benefit the guardians instead of the governor here tbh, I put it here cause it wasn't shared between other modules